lib/file.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import hashlib
+
+
+class FileSystem:
```

Comment:
I'm dubious you need a class for this but ok ...

it seems a simple method would be sufficient:

```
get_file_info(file_path) => dict
```